earth_enterprise/tutorial/FusionTutorial/download_tutorial.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+TMPFILE=/tmp/FusionTutorial-Full.tar.gz
+SCRIPTDIR=`dirname $0`
```

Comment:
I think this demonstrates what @tst-ppenev means:
```
ccamp@xtree-ubuntu:/$ cat /tmp/test\ me/testbash.sh 
#!/bin/bash
SCRIPTDIR=`dirname "$0"`
SCRIPTDIR2=`dirname $0`
echo $SCRIPTDIR
echo $SCRIPTDIR2

ccamp@xtree-ubuntu:/$ /tmp/test\ me/testbash.sh 
/tmp/test me
/tmp me
```

The first line is the correct way to use quotes so that it will pick up weird directories, such as ones that contain spaces. Anywhere you use $0 or $SCRIPTDIR you'll surround it with double quotes. FYI, you don't want to use single quotes because that makes the contents literal. i.e. it will not resolve variables and give you the variable name instead.